infra/examples-dev/gcp/main.tf

```
@@ -152,7 +152,8 @@ output "todos_1" {
   value = var.todos_as_outputs ? join("\n",
     concat(
       module.worklytics_connectors.todos,
-      module.worklytics_connectors_google_workspace.todos
+      module.worklytics_connectors_google_workspace.todos,
+      ""
```

this does't seem correct to me .. `concat()` in terraform is a list function

https://developer.hashicorp.com/terraform/language/functions/concat
```suggestion
      []
```